Create README.md (#128)
Laura-Novich-OBW/student-showcase: student-work/2022/Debbie-Mankowitz/api-final-project/README.md

# Hamburger API DOC

> This is the new APP for customers for online orders for General Putnam Motel Diner (GPMD). 

Reviewer: ```suggestion
This API is for creating a new APP for General Putnam Motel Diner (GPMD).  Customers of GPMD can use the app to order food. This initial offering is a Proof of Concept (PoC). 
```

Reviewer: Debbie if you like the change I am suggesting click the "Commit Suggestion" button and commit the change.

> This App is intended for customers as a preliminary version for take-out hamburger meals.

Reviewer: ```suggestion
It is limited in scope and the preliminary version only supports creating a take-out hamburger meal order.
```

> If the App becomes popular and easy to use then GPMD will extend their APP to include other offerings on their menu.

Reviewer: ```suggestion
If the PoC is accepted, the App can be expanded so that GPMD can extend their APP to include other offerings on their menu.
```
